cli/command/container/opts.go

```diff
@@ -485,6 +485,11 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 		return nil, err
 	}
 
+	securityOpts, maskedPaths, readonlyPaths, err := parseSystemPaths(securityOpts)
+	if err != nil {
```

Comment:
I see we never return an error; although it's good practice to keep this signature, this function is not exported, so I think it's ok to drop the `error` return for now, and remove this error-handling